packages/translate-results/lib/__tests__/progress.test.ts

```diff
@@ -0,0 +1,40 @@
+import {translateProgress} from '../progress';
+
+describe('Interactive progress explanation.', () => {
+	describe('Test actionFailed status with codes.', () => {
+		it('Should return empty result if code not provided', () => {
+			expect(translateProgress({status: 'actionFailed'}))
+				.toStrictEqual({title: ''});
+		});
+
+		it('Should return empty result if code is unknown', () => {
+			expect(translateProgress({status: 'actionFailed', code: 'unknownCode' as any}))
+				.toStrictEqual({title: ''});
+		});
+
+		it('Should return translation if code is valid', () => {
+			expect(translateProgress({code: 'noActivePlan'}))
+				.toStrictEqual({
+					title: 'Cannot continue: Your subscription has expired',
```

Comment:
We can add snapshot tests for all translations

Reply:
I like this idea